webhook_server/tests/test_github_api_metrics.py

```
@@ -280,8 +308,13 @@ async def test_get_token_metrics_fallback_reset(
         assert "initial: 100" in metrics
         assert "final: 5000" in metrics
 
-    def test_del_safe_missing_attr(self, minimal_hook_data, minimal_headers, logger):
-        # Create instance but don't init fully to avoid setting attributes
+    def test_del_cleanup_without_clone_dir(self, minimal_hook_data, minimal_headers, logger):
```

Ruff (0.15.12)

[warning] Unused method argument: `minimal_hook_data`

(ARG002)